plugins/ftdidmx/FtdiWidget.cpp

```diff
@@ -364,14 +372,35 @@ bool FtdiInterface::Write(const ola::DmxBuffer& data) {
   }
 }
 
-bool FtdiInterface::Read(unsigned char *buff, int size) {
+
+bool FtdiInterface::Write(ola::io::ByteString *packet) {
+  int bytesWritten = ftdi_write_data(
+                       &m_handle,
+                       const_cast<unsigned char *>(packet->data()),
+                       packet->size());
+
+  int size = packet->size();
+  if (bytesWritten < 0) {
+    OLA_WARN << m_parent->Description() << " "
+             << ftdi_get_error_string(&m_handle);
+    return false;
+  } else if (bytesWritten == static_cast<int>(packet->size())) {
+    return true;
+  } else {
+    OLA_WARN << "Bytes Written: " << bytesWritten << " Packet Size: " << size;
```

Comment:
Can you make this more explicit, add a != in the log text.

Reply:
Fixed